inc/cappuccino/TlruCache.h

```diff
@@ -55,16 +55,29 @@ class TlruCache {
         size_t capacity,
         float max_load_factor = 1.0f);
 
+    /**
+     * Inserts the given key value pair with the new TTL, but only if key doesn't already exist or is expired.
+     * @param ttl The TTL for this key value pair.
+     * @param key The key to store the value under.
+     * @param value The value of data to store.
+     * @return  True if new key inserted (not found or expired), False if already exists and not inserted
+     */
+    auto InsertOnly(
```

Comment:
It kind of seems like this should be `Insert `or `InsertOnly` and `Insert` should be `InsertOrUpdate`.

Reply:
Yeah -- I was avoiding changing the original interface. Think I should change it?

Reply:
Right, good point. Not changing it is probably best.